type-morph/main.ts

```
@@ -171,6 +183,45 @@ function parseImportLine(line: string, language: string): { module: string; item
+        const packageName = parts[parts.length - 1];
+        return { module, items: [packageName] };
+      }
+      const goAliasMatch = line.match(/^import\s+(\S+)\s+"([^"]+)"$/);
+      if (goAliasMatch) {
+        const alias = goAliasMatch[1];
+        const module = goAliasMatch[2];
+        return { module, items: [alias] };
+      }
+      break;
+    case 'swift':
+      const swiftMatch = line.match(/^import\s+(\S+)$/);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)